coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorConf.java

```diff
@@ -47,6 +47,12 @@ public class CoordinatorConf extends RssBaseConf {
       .longType()
       .defaultValue(30 * 1000L)
       .withDescription("timeout if can't get heartbeat from shuffle server");
+  public static final ConfigOption<Long> COORDINATOR_NODES_PERIODIC_OUTPUT_INTERVAL_TIMES = ConfigOptions
+      .key("rss.coordinator.server.periodic.output.interval.times")
+      .longType()
```

Comment:
Could we add `checkValue`?

Reply:
Yes. I will add check.

Reply:
Could we add some the document?

Reply:
It's OK. But currently the file of doc/coordinator_guide.md is empty and the README about coordinator conf is  missing lots of params.

So do i add this conf into README?

Reply:
Could you migrate the cooridinator's document from README to doc/coordinator_guide.md  and add the document of this config option in the doc/coordinator_guide.md in a new pr?

Reply:
Got it. I'm glad to do this.

Reply:
OK, let's merge this pr first.